modules/svg/defs.js

```diff
@@ -13,18 +13,6 @@ export function svgDefs(context) {
     function drawDefs(selection) {
         var defs = selection.append('defs');
 
-        // add pattern for AI buildings
-        defs.append('pattern')
-            .attr('id', 'ai-building')    
-            .attr('width', 4)
-            .attr('height', 15)
-            .attr('patternUnits', 'userSpaceOnUse')
-            .attr('patternTransform', 'rotate(45 50 50)')
-            .append('line')
-            .attr('stroke', '#ff26d4')
-            .attr('stroke-width', '2px')
-            .attr('y2', '15'); 
-    
```

Comment:
Hahaha yes!  They are still there and still beautiful, I've just moved them so that now the Rapid features have their own `<defs>` where it stores all the Rapid patterns.  And I generate a unique diagonal stroke for each dataset:

https://github.com/bhousel/RapiD/blob/6e86af0b1602917a4671868e72587c111df1bf55/modules/svg/ai_features.js#L171-L190

So we can show different datasets on top of one another:

![Screenshot from 2020-05-08 11-51-15](https://user-images.githubusercontent.com/38784/82356956-56671700-99d2-11ea-8067-443c586d7a4d.png)

BTW: Maybe we can have even more fun with patterns!  Check these out:  https://twitter.com/presentcorrect/status/1260952522686070787